components/form-builder/app/edit/Edit.tsx

```diff
@@ -72,7 +72,13 @@ export const Edit = () => {
       <RichTextLocked
         beforeContent={
           <>
-            <label htmlFor="formTitle" className="visually-hidden">
+            <label
+              htmlFor="formTitle"
+              className="visually-hidden"
+              {...(i18n.language !== translationLanguagePriority && {
```

Comment:
Something that might be worth trying 


Setup a store helper something like:

```
useTemplateStore
...
getLocalizationLang: () => get().lang !== get().translationLanguagePriority ? translationLanguagePriority : "",
```

lang in the store should be in sync with i18n.language and this would avoid having the same check in multiple spots if we need to update.

Reply:
Makes sense. Were you thinking something like placing a function like this for re-use at the top of files like Edit.tsx or more a util like function that would go in something like useTemplateStore.tsx? 
The second, useTemplateStore, I might need some guidance. I skimmed it and most files in the Form Builder but that one I think understand but not 100% :)

Reply:
A helper in useTemplateStore.tsx --- that we can pull in via useTemplateStore

Reply:
A helper in useTemplateStore.tsx --- that we can pull in via useTemplateStore

Reply:
Thinking something like https://github.com/cds-snc/platform-forms-client/commit/bde632bf72a3430abad122cf00086d8ed45a3b32

Reply:
Looks good, I'll add/merge that